Orange/widgets/data/owtranspose.py

```diff
@@ -76,25 +106,40 @@ def update_controls(self):
             if self.feature_model:
                 self.feature_names_column = self.feature_model[0]
         enabled = bool(self.feature_model)
-        self.feature_radio.buttons[1].setEnabled(enabled)
+        self.feature_radio.buttons[self.FROM_META_ATTR].setEnabled(enabled)
```

Comment:
The previous version allowed changing the radio button, but prevented changing the combo when no data. I don't see much point in disabling the combo and lineedit, and leave the radios on.

On the other hand, is there any harm in letting the user monkey around the controls when there is no data? I'd just leave them enabled.